src/apps/app/App.ts

```diff
@@ -20,7 +21,19 @@ const resolve = [
       .then(apolloRes => {
         pageResolver({ bundleName: 'app', apolloRes, res })
       })
-      .catch(next)
+      .catch(err => {
+        const STATUS_CODE = getFirstStatusCode(err)
+
+        if (STATUS_CODE === 'UNAUTHORIZED' && req.url === '/') {
+          // This typically happens if the serialized user is "bad"
+          // or not actually logged in. If so: logout, then redirect somewhere.
+          // Falling through by using `next()` doesn't seem to actually purge the session.
+          req.logout()
+          return res.redirect('/log_in')
+        }
+
+        return next(err)
+      })
```

Comment:
Not so sure this makes sense: wouldn't this log you out if you just happened to land on a private channel somehow?

Reply:
There’s a conditional there: 
`if req.url == '/' ‘